### ISSUE TITLE
link to releases does not contain any releases

### ISSUE THREAD

**Issue by inphobia** [Cellule/dndGenerator#204]
https://github.com/Cellule/dndGenerator/blob/0ca03d806190ba070abd74a43ca784d154331d51/src/Footer.tsx#L71C1-L71C84

heya,
while npcgenerator.com has a link to github releases, there aren't any releases availlable. perhaps you could add a git tag and make that available as a release?

i checked the commit history and it seems 0.5.0 came from this commit; 11f0dd59295c356c86d733001aec89a6e0607867
with the addemdum of this commit: 1bb0f9f027d2487974a9beb88393a09636cd319f

or perhaps you can just tag master since, as far a i can tell, the version number currently isn't used? (perhaps somewhere in node/npm, dunno how that publishing workflow goes).

guess you'll still need to manually make a release bundle after adding the tag: https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository?tool=webui

and thank you for this wonderfull waste of time, love the funky plot hooks.

**Comment by Cellule** [comment]
Good point, I forgot we had a link to the Releases page on the footer. 
We don't really use releases and there's not much active development at this time, mainly keeping thing up-to-date and running.
I'd like to move to a CHANGELOG.md file instead and likely point to that instead of the Releases page.
I'll try to look into that when I have some time

**Comment by inphobia** [comment]
thought i'd give a hand and tag it for you, but can't find a way make a pull request for only a tag.


to retroactively add it you can use:
```
git tag -a 0.5.0 1bb0f9f027d2487974a9beb88393a09636cd319f
git add .
git commit
git push
```

this will add a tarball labeled 0.5.0 under https://github.com/Cellule/dndGenerator/tags
... and doesn't completely fix this issue since that's still not https://github.com/Cellule/dndGenerator/releases :)


to fix that you'll need to checkout the above mentioned commit, update footer, add tag, push it to a new branch:
i don't recommend doing this....
```
git checkout 1bb0f9f027d2487974a9beb88393a09636cd319f -b t050
vi ./src/Footer.tsx
git add .
git commit
git tag -a 0.5.0
git push
```


perhaps it's easier just to:
 * update footer. example https://github.com/Cellule/dndGenerator/compare/master...inphobia:dndGenerator:t050
 * change version to 0.5.1 in package-lock.json & package.json
 * git add, git commit
 * git tag -a 0.5.1
 * push to github


your proposal of moving to a changelog solves it as well. imo however, for reference & dependencies a tag or release is easier to work with since it's a fixed point.

whatever works for you.

**Comment by Cellule** [comment]
Took me forever, but I finally made a "proper" release. It's not the greatest thing since I lost track of what changed.
Indeed the versions are not "super" relevant, I was mainly using it for big overhaul (which I'm kinda due again)
At least now the link on the website leads to something.
I'll try to update this in future changes and hopefully keep it relevant.

Thanks again!
